staff/gerard-celma/precourse/tema8pasapalabrapro/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="es-ES">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>PASAPALABRA</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" media="all">
+    <link href="https://fonts.googleapis.com/css?family=Lato:700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Comfortaa:700" rel="stylesheet">
+    <script src="/script.js" async></script>
```

Comment:
You have to remove the / from the script.js because the program can find your js file. 
It has to look like <script src="script.js" async></script>

Reply:
Also it will be nice that the text input became empty every time the user pass or response

Reply:
script link fixed, second bug will be more difficult ;)